### ISSUE TITLE
Plantejament: debatim si fem i què fem? :-)

### ISSUE THREAD

**Issue by calbasi** [calbasi/Manual-Cyberprivacitat#1]
Penjo aquí la conversa de Mastodont que dona origen al repo:
https://todon.nl/@arbocenc/99496931051481959

**Comment by calbasi** [comment]
Si us sembla bé, jo obriria dos issues/temes de debat: 

1) Quin hauria de ser l'abast de la "guia"
2) Quines funcionalitats volem que tingui i, per tant, quina eina/es usem per fer-les possibles.

Va bé?

**Comment by jprojects** [comment]
Si, bé jo crec que en principi hauríem de donar a conèixer eines/comportaments segurs per la privacitat a la xarxa.

Perquè tothom ho entengui potser una FAQ amb preguntes respostes seria una solució.

Format: Un web amb html amb desplazament horitzontal?, seria maco amb una pàgina de cada color amb una pregunta habitual i la resposta nostre acompanyat d'una imatge d'un personatge fent l'acció.

És una primera idea.

**Comment by calbasi** [comment]
Ei, el que jo volia fer ja està fet. Aquí tens la traducció al castellà:
https://victorhck.gitlab.io/privacytools-es
Llavors, hi ha altres coses que es poden fer: per exemple:
- que tothom pugui passar un checklist amb el que fa o encara no fa, i l'aplicació li digui els punts forts/febles.
- que cadascuna d'aquestes passes incorpori un how-to, adreçat a usuaris d'a peu... Però potser això no és massa lògic, és reinventar la roda, perquè el senzill és adreçar la gent a les pàgines de cada "sol·lució" a on probablement ja els expliquen com usar-la millor que ningú.
- muntar xerrades locals a on tocar alguns (pocs) d'aquests punts, una mena de xerrades pràctiques a on la gent porta els seus dispositius. Segur que una part d'aquesta feina ja es fa via hacklabs o ves a saber... La qüestió és que amb el conflicte polític molta gent pot estar-hi interessada (d'altra manera tots aquests temes de la privacitat li rellisquen al 99% de la penya)

**Comment by calbasi** [comment]
Aquí hi ha el projecte original: https://github.com/privacytoolsIO/privacytools.io
